lib/perl/Genome/Process.pm

```diff
@@ -710,19 +710,28 @@ sub _compare_output_directories {
                 my $rel_path = File::Spec->abs2rel($file, $output_dir);
                 $diffs{$rel_path} = sprintf($template, $type, $rel_path, $other_process->id);
             } else {
-                if (-d $target && -d $other_target) {
-                    my %additional_diffs = $self->_compare_output_directories($target, $other_target, $other_process);
-                    %diffs = (%diffs, %additional_diffs);
-                }
-                elsif (-f $target && -f $other_target && !compare($target, $other_target)) {
-                    #Files are in fact the same - do nothing
-                }
-                else {
-                    $diffs{File::Spec->abs2rel($file, $output_dir)} = sprintf(
-                        'files are not the same (diff -u %s %s)',
-                        $file, $other_file
-                    );
+                if (-l $file || -l $other_file) {
```

Comment:
Is that based on an assumption that this will always be called by `_compare_output_files`?